deploy/deploy.yaml

```diff
@@ -1903,7 +1920,7 @@ spec:
         env:
         - name: WATCH_NAMESPACE
           value: ""
-        image: docker.io/solace/pubsubplus-eventbroker-operator:1.0.3
+        image: ghcr.io/solacedev/pubsubplus-eventbroker-operator:1.2.0
```

Comment:
is this required as part of your change?

Reply:
Yeah, the release points to users to the dockerhub for the latest image but during development we use GHCR as the images change and we only push to dockerhub as part of the release.